{"timestamp":"2025-05-21T06:18:00.229731","level":"info","event":"DAG bundles loaded: dags-folder","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-05-21T06:18:00.239017","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/load_data.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-05-21T06:18:04.313466","level":"info","event":"Secrets backends loaded for worker","count":1,"backend_classes":["EnvironmentVariablesBackend"],"logger":"supervisor"}
{"timestamp":"2025-05-21T06:18:04.672996","level":"info","event":"Connection Retrieved 'snowflake-connect'","logger":"airflow.hooks.base"}
{"timestamp":"2025-05-21T06:18:04.675844","level":"info","event":"Snowflake Connector for Python Version: 3.15.0, Python Version: 3.12.10, Platform: Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.36","logger":"snowflake.connector.connection"}
{"timestamp":"2025-05-21T06:18:04.678372","level":"info","event":"Connecting to GLOBAL Snowflake domain","logger":"snowflake.connector.connection"}
{"timestamp":"2025-05-21T06:18:10.744232","level":"warning","event":"Certificate did not match expected hostname: bzmmwmw.singapore.snowflakecomputing.com. Certificate: {'subject': ((('commonName', '*.prod3.us-west-2.snowflakecomputing.com'),),), 'subjectAltName': [('DNS', '*.prod3.us-west-2.snowflakecomputing.com'), ('DNS', '*.us-west-2.snowflakecomputing.com'), ('DNS', '*.global.snowflakecomputing.com'), ('DNS', '*.snowflakecomputing.com'), ('DNS', '*.prod3.us-west-2.aws.snowflakecomputing.com')]}","logger":"snowflake.connector.vendored.urllib3.connection"}
{"timestamp":"2025-05-21T06:18:10.745887","level":"warning","event":"Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(CertificateError(\"hostname 'bzmmwmw.singapore.snowflakecomputing.com' doesn't match either of '*.prod3.us-west-2.snowflakecomputing.com', '*.us-west-2.snowflakecomputing.com', '*.global.snowflakecomputing.com', '*.snowflakecomputing.com', '*.prod3.us-west-2.aws.snowflakecomputing.com'\"))': /session/v1/login-request?request_id=7ccc51da-1bcf-4467-a6aa-0ca6b1b4c284&databaseName=EDW&schemaName=Bronze&warehouse=COMPUTE_WH&roleName=ACCOUNTADMIN&request_guid=2a4c18b2-fb7a-45c6-90b0-9fff5b7a2c41","logger":"snowflake.connector.vendored.urllib3.connectionpool"}
{"timestamp":"2025-05-21T06:18:11.580069","level":"warning","event":"Certificate did not match expected hostname: bzmmwmw.singapore.snowflakecomputing.com. Certificate: {'subject': ((('commonName', '*.prod3.us-west-2.snowflakecomputing.com'),),), 'subjectAltName': [('DNS', '*.prod3.us-west-2.snowflakecomputing.com'), ('DNS', '*.us-west-2.snowflakecomputing.com'), ('DNS', '*.global.snowflakecomputing.com'), ('DNS', '*.snowflakecomputing.com'), ('DNS', '*.prod3.us-west-2.aws.snowflakecomputing.com')]}","logger":"snowflake.connector.vendored.urllib3.connection"}
{"timestamp":"2025-05-21T06:18:13.599616","level":"warning","event":"Certificate did not match expected hostname: bzmmwmw.singapore.snowflakecomputing.com. Certificate: {'subject': ((('commonName', '*.prod3.us-west-2.snowflakecomputing.com'),),), 'subjectAltName': [('DNS', '*.prod3.us-west-2.snowflakecomputing.com'), ('DNS', '*.us-west-2.snowflakecomputing.com'), ('DNS', '*.global.snowflakecomputing.com'), ('DNS', '*.snowflakecomputing.com'), ('DNS', '*.prod3.us-west-2.aws.snowflakecomputing.com')]}","logger":"snowflake.connector.vendored.urllib3.connection"}
{"timestamp":"2025-05-21T06:18:13.603430","level":"warning","event":"Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(CertificateError(\"hostname 'bzmmwmw.singapore.snowflakecomputing.com' doesn't match either of '*.prod3.us-west-2.snowflakecomputing.com', '*.us-west-2.snowflakecomputing.com', '*.global.snowflakecomputing.com', '*.snowflakecomputing.com', '*.prod3.us-west-2.aws.snowflakecomputing.com'\"))': /session/v1/login-request?request_id=1bdc2874-0892-474c-82f6-35f51be94bf3&databaseName=EDW&schemaName=Bronze&warehouse=COMPUTE_WH&roleName=ACCOUNTADMIN&request_guid=a7867245-61f8-48fb-b48d-749aa3317336","logger":"snowflake.connector.vendored.urllib3.connectionpool"}
{"timestamp":"2025-05-21T06:18:14.453148","level":"warning","event":"Certificate did not match expected hostname: bzmmwmw.singapore.snowflakecomputing.com. Certificate: {'subject': ((('commonName', '*.prod3.us-west-2.snowflakecomputing.com'),),), 'subjectAltName': [('DNS', '*.prod3.us-west-2.snowflakecomputing.com'), ('DNS', '*.us-west-2.snowflakecomputing.com'), ('DNS', '*.global.snowflakecomputing.com'), ('DNS', '*.snowflakecomputing.com'), ('DNS', '*.prod3.us-west-2.aws.snowflakecomputing.com')]}","logger":"snowflake.connector.vendored.urllib3.connection"}
{"timestamp":"2025-05-21T06:18:14.477464","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"OperationalError","exc_value":"250001: 250001: Could not connect to Snowflake backend after 2 attempt(s).Aborting\nVerify that the hostnames and port numbers in SYSTEM$ALLOWLIST are added to your firewall's allowed list.\nTo further troubleshoot your connection you may reference the following article: https://docs.snowflake.com/en/user-guide/client-connectivity-troubleshooting/overview.","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":838,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1130,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":408,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":212,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":235,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"},{"filename":"/opt/airflow/dags/load_data.py","lineno":13,"name":"test_snowflake_connection"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py","lineno":589,"name":"get_records"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py","lineno":515,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py","lineno":383,"name":"get_conn"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/__init__.py","lineno":54,"name":"Connect"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/connection.py","lineno":548,"name":"__init__"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/connection.py","lineno":901,"name":"connect"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/connection.py","lineno":1293,"name":"__open_connection"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/connection.py","lineno":1615,"name":"authenticate_with_retry"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/connection.py","lineno":1693,"name":"_authenticate"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/connection.py","lineno":1668,"name":"_authenticate"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/auth/by_plugin.py","lineno":211,"name":"handle_timeout"}]},{"exc_type":"OperationalError","exc_value":"250003: 250003: Hit non-retryable SSL error, HTTPSConnectionPool(host='bzmmwmw.singapore.snowflakecomputing.com', port=443): Max retries exceeded with url: /session/v1/login-request?request_id=7ccc51da-1bcf-4467-a6aa-0ca6b1b4c284&databaseName=EDW&schemaName=Bronze&warehouse=COMPUTE_WH&roleName=ACCOUNTADMIN&request_guid=2a4c18b2-fb7a-45c6-90b0-9fff5b7a2c41 (Caused by SSLError(CertificateError(\"hostname 'bzmmwmw.singapore.snowflakecomputing.com' doesn't match either of '*.prod3.us-west-2.snowflakecomputing.com', '*.us-west-2.snowflakecomputing.com', '*.global.snowflakecomputing.com', '*.snowflakecomputing.com', '*.prod3.us-west-2.aws.snowflakecomputing.com'\"))).\nVerify that the hostnames and port numbers in SYSTEM$ALLOWLIST are added to your firewall's allowed list.\nTo further troubleshoot your connection you may reference the following article: https://docs.snowflake.com/en/user-guide/client-connectivity-troubleshooting/overview.","exc_notes":[],"syntax_error":null,"is_cause":true,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/connection.py","lineno":1647,"name":"_authenticate"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/auth/_auth.py","lineno":226,"name":"authenticate"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/network.py","lineno":759,"name":"_post_request"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/network.py","lineno":874,"name":"fetch"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/network.py","lineno":1002,"name":"_request_exec_wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/network.py","lineno":924,"name":"_request_exec_wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/network.py","lineno":1161,"name":"_request_exec"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/errors.py","lineno":279,"name":"errorhandler_wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/errors.py","lineno":337,"name":"hand_to_other_handler"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/errors.py","lineno":210,"name":"default_errorhandler"}]},{"exc_type":"SSLError","exc_value":"HTTPSConnectionPool(host='bzmmwmw.singapore.snowflakecomputing.com', port=443): Max retries exceeded with url: /session/v1/login-request?request_id=7ccc51da-1bcf-4467-a6aa-0ca6b1b4c284&databaseName=EDW&schemaName=Bronze&warehouse=COMPUTE_WH&roleName=ACCOUNTADMIN&request_guid=2a4c18b2-fb7a-45c6-90b0-9fff5b7a2c41 (Caused by SSLError(CertificateError(\"hostname 'bzmmwmw.singapore.snowflakecomputing.com' doesn't match either of '*.prod3.us-west-2.snowflakecomputing.com', '*.us-west-2.snowflakecomputing.com', '*.global.snowflakecomputing.com', '*.snowflakecomputing.com', '*.prod3.us-west-2.aws.snowflakecomputing.com'\")))","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/network.py","lineno":1091,"name":"_request_exec"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/vendored/requests/sessions.py","lineno":589,"name":"request"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/vendored/requests/sessions.py","lineno":703,"name":"send"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/vendored/requests/adapters.py","lineno":516,"name":"send"}]},{"exc_type":"MaxRetryError","exc_value":"HTTPSConnectionPool(host='bzmmwmw.singapore.snowflakecomputing.com', port=443): Max retries exceeded with url: /session/v1/login-request?request_id=7ccc51da-1bcf-4467-a6aa-0ca6b1b4c284&databaseName=EDW&schemaName=Bronze&warehouse=COMPUTE_WH&roleName=ACCOUNTADMIN&request_guid=2a4c18b2-fb7a-45c6-90b0-9fff5b7a2c41 (Caused by SSLError(CertificateError(\"hostname 'bzmmwmw.singapore.snowflakecomputing.com' doesn't match either of '*.prod3.us-west-2.snowflakecomputing.com', '*.us-west-2.snowflakecomputing.com', '*.global.snowflakecomputing.com', '*.snowflakecomputing.com', '*.prod3.us-west-2.aws.snowflakecomputing.com'\")))","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/vendored/requests/adapters.py","lineno":485,"name":"send"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/vendored/urllib3/connectionpool.py","lineno":827,"name":"urlopen"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/vendored/urllib3/connectionpool.py","lineno":799,"name":"urlopen"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/vendored/urllib3/util/retry.py","lineno":592,"name":"increment"}]},{"exc_type":"CertificateError","exc_value":"hostname 'bzmmwmw.singapore.snowflakecomputing.com' doesn't match either of '*.prod3.us-west-2.snowflakecomputing.com', '*.us-west-2.snowflakecomputing.com', '*.global.snowflakecomputing.com', '*.snowflakecomputing.com', '*.prod3.us-west-2.aws.snowflakecomputing.com'","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/vendored/urllib3/connectionpool.py","lineno":715,"name":"urlopen"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/vendored/urllib3/connectionpool.py","lineno":404,"name":"_make_request"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/vendored/urllib3/connectionpool.py","lineno":1058,"name":"_validate_conn"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/vendored/urllib3/connection.py","lineno":472,"name":"connect"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/vendored/urllib3/connection.py","lineno":545,"name":"_match_hostname"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/vendored/urllib3/util/ssl_match_hostname.py","lineno":150,"name":"match_hostname"}]}]}
